samples/IISSample/web.config

```diff
@@ -1,9 +0,0 @@
-﻿<?xml version="1.0"?>
```

Comment:
No, keep the tooling. The sample should resemble the templates, which contain a web.config with environment variables.